rack-protection/lib/rack/protection/http_origin.rb

```diff
@@ -32,7 +32,14 @@ def accepts?(env)
         return true unless origin = env['HTTP_ORIGIN']
         return true if base_url(env) == origin
         return true if options[:allow_if] && options[:allow_if].call(env)
-        Array(options[:origin_whitelist]).include? origin
+
+        if options.key? :origin_whitelist
+          warn "Rack::Protection origin_whitelist option is deprecated and will be removed, " \
+            "use origin_whitelist instead.\n"
```

Comment:
@rhymes just realized this warning message needs tweaked 😄

Reply:
ahahaha great catch!